src/DataTable/Table.js

```diff
@@ -65,7 +65,7 @@ class Table extends React.Component {
                 </thead>
                 <tbody>
                     {realRows.map((row, idx) =>
-                        <tr key={row[rowKeyColumn] || row.key || idx} className={row.className}>
+                        <tr key={row[rowKeyColumn] || row.key || idx} className={row.className} style={row.style}>
```

Comment:
Hmm...

I know we're using that for `row.className` but I wonder if with `style` it could produce errors. Because if the rows data have `style` as a key, and that style is a string for example, React will not be happy.

I wonder if we could find a better way to allow the row styling (even for the className to be honest).